### PR TITLE
SimplexMQ readme, remove chat client

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,223 +1,67 @@
...
-- no support for chat groups. It is coming in the next major version (i.e., not very soon:)
-- no delivery notifications - coming soon
-- no TCP transport encryption - coming soon (messages are encrypted e2e though, only random connection IDs and server commands are visible, but not the contents of the message)
-- system and protocol security was not audited yet, so you probably should NOT use it yet for high security communications - unless you know what you are doing.
+## SimpleXMQ roadmap
 
-## How to run chat client locally
+- Streams - high performance message queues. See [Streams RFC](./rfcs/2021-02-28-streams.md) for details.
+- "Small" connection groups, when each message will be sent by the SMP agent to multiple connections with a single client command. See [Groups RFC](./rfcs/2021-03-18-groups.md) for details.
+- SMP agents cluster to share connections and message management by multiple agents.
```
Comment (efim-poberezkin) [inline README.md:18]: I had to spend some time processing this - translate for people it would allow multi-device for simplex-chat?

Comment (efim-poberezkin) [reply README.md:18]: I understand it's only a subset of possible uses, but still

Comment (epoberezkin) [reply README.md:18]: Yes

File: README.md [hunk 2]
```diff
@@ -1,223 +1,67 @@
...
-The party that received the invitation should use `/accept <name2> <invitation>` to accept the connection (`<name2>` is any name that the accepting party wants to use for you).
+![SMP server design](./design/server.svg)
 
-For example, if Alice and Bob want to chat, with Alice initiating, Alice would use [in her chat client]:
+## SMP agent design
 
-```
-/add bob
-```
+![SMP agent design](./design/agent2.svg)
```
Comment (efim-poberezkin) [inline README.md:63]: there's a `Q: can multiple threads use it` not on sqlite block - shall we remove it? I mean we know they can.. with caveats

File: README.md [hunk 3]
```diff
@@ -1,223 +1,67 @@
...
-```
+SMP server implements [SMP protocol](./protocol/simplex-messaging.md).
 
-> **NOTE:** When running chat client executable built with the latter approach, if you encounter ``version `GLIBC_2.28' not found`` error, rebuild it with `haskell:8.8.4-stretch` base image instead (you'd have to change it in your local [Dockerfile](Dockerfile)).
+### SMP client library
 
-`dog-food` (as in "eating your own dog food" - it is an early prototype) starts chat client with default parameters. By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and SQLite database file `smp-chat.db` is initialized in it. The default SMP server is `smp.simplex.im:5223`.
+[SMP client](./src/Simplex/Messaging/Client.hs) is a Haskell library to connect to SMP servers that allows to:
+- execute commands with a functional API.
+- receive message and other notifications via STM queue.
```
Comment (efim-poberezkin) [inline README.md:38]: message -> messages? or I'm reading it strangely..

**Review by efim-poberezkin — APPROVED** [review]

**Pull request merged**